Tasks/InstallAppleProvisioningProfileV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,6 +1,6 @@
 {
   "loc.friendlyName": "Install Apple provisioning profile",
-  "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=862068)",
+  "loc.helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=862068)",
```

Comment:
Are these intentional changes?

Reply:
Same as above - autogenerated by the build